CMakeLists.txt

```diff
@@ -1,6 +1,6 @@
 cmake_minimum_required(VERSION 3.12.0 FATAL_ERROR)
 
-option(IRODS_ENABLE_430_COMPATIBILITY "Enables compatibility with iRODS 4.3.0." YES) # Default to YES for community.
+option(IRODS_ENABLE_430_COMPATIBILITY "Enables compatibility with iRODS 4.3.0." NO) # Default to YES for community.
```

Comment:
Tell me about this?

Reply:
This wasn't supposed to make it into the PR, will fix